docs/source/EM_analyzers/stator_wdg_res.rst

```diff
@@ -1,42 +1,61 @@
 Stator Winding Resistance Analyzer
 ##########################################
 
-This analyzer determines stator winding resistance, considering effects of end windings.
+This analyzer calculates stator winding resistance and coil length, considering effects of end windings.
 
 Model Background
 ****************
 
-The following equation is used to calculate stator winding resistance:
+This analyzer implements the coil length and resistance calculation approach provided in the following reference:
+
+* N. Bianchi, S. Bolognani, and P. Frare, “Design Criteria for High-Efficiency SPM Synchronous Motors,” in `IEEE Transactions on Energy Conversion`, June 2006.
+
+The calculation approach is now summarized. The following equation is used to calculate stator winding resistance:
 
 .. math::
 
     R_\text{wdg} &= \frac{z_Q z_C l_\text{coil}}{\sigma_\text{cond} A_\text{cond}}\\
 
-where :math:`z_Q` is a number of turns per coil, :math:`z_C` is a number of coils per phase, :math:`l_\text{coil}` is a length of a coil including coil ends, and :math:`\sigma_\text{cond}` and :math:`A_\text{cond}` are conductor conductivity and cross-section area. 
-Conductor area is found as :math:`A_\text{cond} = K_\text{Cu}A_\text{slot}/z_Q`, where :math:`A_\text{slot}` is a stator slot area and :math:`K_\text{Cu}` is a slot fill factor. The length of a coil consists of two main parts:
+where :math:`z_Q` is a number of turns per coil, :math:`z_C` is a number of coils per phase, :math:`l_\text{coil}` is an average length of a single loop of a coil, and :math:`\sigma_\text{cond}` and :math:`A_\text{cond}` are conductor conductivity and cross-section area. 
+Conductor area is found as :math:`A_\text{cond} = K_\text{Cu}A_\text{slot}/(n_\text{layers}z_Q)`, where :math:`A_\text{slot}` is a stator slot area, :math:`K_\text{Cu}` is a slot fill factor, and :math:`n_\text{layers}` is a number of layers (:math:`n_\text{layers}=1` for a single-layer winding and :math:`n_\text{layers} = 2` for a double-layer winding).
```

Comment:
To increase readability, let's make this equation be free standing like the `Rwdg` equation
<img width="228" alt="image" src="https://github.com/Severson-Group/eMach/assets/7421348/1525482a-3a24-4886-a443-1fae2d52841f">

i.e., make it look like this:
<img width="191" alt="image" src="https://github.com/Severson-Group/eMach/assets/7421348/9e012c9e-641c-44d2-857c-d39aa01c4451">